app/controllers/ChoiceController.scala

```diff
@@ -89,4 +88,23 @@ class ChoiceController @Inject()(
       )
   }
 
+  private def updateChoice(id: String, choice: Choice)(implicit hc: HeaderCarrier) =
+    exportsCacheService.get(id).map(_.map(_.copy(choice = choice.value))).flatMap {
+      case Some(declaration) => exportsCacheService.update(declaration)
+      case None =>
+        logger.error(s"Failed to find declaration for id $id")
+        Future.successful(None)
+    }
+
+  private def create(choice: Choice)(implicit hc: HeaderCarrier) =
+    exportsCacheService
+      .create(
+        ExportsDeclaration(
+          None,
+          DeclarationStatus.DRAFT,
+          createdDateTime = Instant.now,
+          updatedDateTime = Instant.now,
+          choice.value
+        )
```

Comment:
Similar here, I'll add method to `ExportsDeclaration` which will take choice as argument and return this declaration, it's some kind of empty declaration so it can be useful in the future

Reply:
I agree that some sort of factory method to create a new draft dec of a given type would be useful.   But perhaps we could create a ticket to create a new `DecType` value and separate the valid values from 'Choice'.   Seems a bit odd that you can create a declaration from a `continue saved declaration`